easybuild/easyconfigs/b/Boost/Boost-1.66.0-foss-2017b.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Boost'
+version = '1.66.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+checksums = ['bd0df411efd9a585e5a2212275f8762079fed8842264954675a4fddc46cfcf60']
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+]
+
+# Beware: There are bugs in the handling of --with-libraries vs
+# --without-libraries.
+configopts = '--with-libraries=all --without-libraries=python'
```

Comment:
using `-with-libraries=all` probably warrants a version suffix, to distinguish from all the other Boost easyconfigs (?)